### PR TITLE
Implement ThemeChanged for web target.

### PR DESCRIPTION
**Pull request by daxpedda**
- [x] Compilation warnings were addressed
- [x] `cargo fmt` has been run on this branch
- [x] `cargo doc` builds successfully
- [x] Added an entry to `CHANGELOG.md` if knowledge of this change could be valuable to users
- [x] Updated documentation to reflect any user-facing changes, including notes of platform-specific behavior
- [x] Updated [feature matrix](https://github.com/rust-windowing/winit/blob/master/FEATURES.md), if new features were added or implemented

Maybe we don't want to make this target specific and just implement it for the other OS's too?

### CODE REVIEW

File: src/platform_impl/web/stdweb/mod.rs [hunk 1]
```diff
@@ -31,6 +32,14 @@ impl WindowExtStdweb for Window {
     fn canvas(&self) -> CanvasElement {
         self.window.canvas().raw().clone()
     }
+
+    fn is_dark_mode(&self) -> bool {
+        let is_dark_mode = js! {
+            return (window.matchMedia && window.matchMedia("(prefers-color-scheme: dark)").matches)
+        };
```
Comment (ryanisaacg) [inline src/platform_impl/web/stdweb/mod.rs]: Might want to throw a TODO here to upstream these APIs to stdweb

File: src/platform_impl/web/stdweb/canvas.rs [hunk 2]
```diff
@@ -240,6 +241,21 @@ impl Canvas {
...
+    {
+        js! {
+            var handler = @{handler};
+
+            if (window.matchMedia) {
+                window.matchMedia("(prefers-color-scheme: dark)").addListener(function(e) {
+                    handler(event.matches)
+                });
+            }
+        }
```
Comment (ryanisaacg) [inline src/platform_impl/web/stdweb/canvas.rs]: Same as other comment

**Comment by ryanisaacg** [comment]
@daxpedda 

> Maybe we don't want to make this target specific and just implement it for the other OS's too?

We can open a tracking issue for it, but don't worry about it for now.

File: src/platform_impl/web/stdweb/mod.rs (hunk 1 above)
Comment (daxpedda) [reply src/platform_impl/web/stdweb/mod.rs]: Done.

File: src/platform_impl/web/stdweb/canvas.rs (hunk 2 above)
Comment (daxpedda) [reply src/platform_impl/web/stdweb/canvas.rs]: Done.

**Comment by ryanisaacg** [comment]
Thanks so much for the PR!

**Pull request merged**